src/cmd/usb.js

```diff
@@ -219,6 +219,7 @@ module.exports = class UsbCommand {
 			.catch(error => {
 				if (until){
 					return deviceMgr.close()
+						.then(() => delay(1000))
```

Comment:
@sergeuz fyi this fixed the [issues i ran into w/ `e2e` tests](https://github.com/particle-iot/particle-cli/pull/623#issuecomment-1172546695) 😅 🤷‍♂️